integration/tests/ipv6/ipv6_test.go

```diff
@@ -52,6 +55,11 @@ var _ = Describe("(Integration) [EKS IPv6 test]", func() {
 		clusterName := params.NewClusterName("ipv6")
 
 		BeforeSuite(func() {
+			f, err := ioutil.TempFile("", "kubeconfig-")
+			Expect(err).NotTo(HaveOccurred())
+			params.KubeconfigPath = f.Name()
+			params.KubeconfigTemp = true
+
 			clusterConfig = api.NewClusterConfig()
 			clusterConfig.Metadata.Name = clusterName
 			clusterConfig.Metadata.Version = "1.21"
```

Comment:
this is a bit out of scope but would you mind making the addons use `latest` as part of this PR? I'm happy to add this commit!

Reply:
go for it!

Reply:
feel free to merge the PR afterwards

Reply:
done! :) thanks